src/ui/public/vislib/visualizations/point_series/area_chart.js

```diff
@@ -96,7 +96,8 @@ export default function AreaChartFactory(Private) {
 
       function y1(d) {
         const y0 = d.y0 || 0;
-        return yScale(y0 + d.y);
```

Comment:
can you add a comment why we are coercing the value now, and not earlier? Is this due to derivative metric in particular?